### PR TITLE
Adding one KB and updating another

### PR DESCRIPTION
**Pull request by amwatson1s**
Adding a KB for RI strategy and updated partner cloud integration consolidated billing

### CODE REVIEW

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  "title": "Partner Cloud Integration: CenturyLink AWS Reserved Instance Strategy",
+  "date": "2-7-2018",
+  "author": "Andy Watson",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+
+CenturyLink [Cloud Application Manager's](https://www.ctl.io/cloud-application-manager/) Cloud Optimization offers [consolidated billing](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-consolidated-billing/) to make payments easy. Part of the consolidated billing involves CenturyLink's strategy for billing Amazon Web Services [Reserved Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-reserved-instances.html).
```
Comment (srveit) [inline Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Change second link to `(partner-cloud-integration-consolidated-billing.md)`

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+### Audience
+
+Users of accounts that already have or are considering CenturyLink Optimization of AWS Accounts. This is for all new standard AWS accounts created through Cloud Application Manager and any existing account moving into CenturyLink's care who have bought Amazon Web Services Reserved Instances.
+
+
+### Prerequisites
+
+For a new AWS Account:
+
+* The customer must have reviewed the process for creating a [new Amazon Web Services account](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-new/) and is thinking about buying AWS Reserved Instances.
```
Comment (srveit) [inline Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Change link to `(partner-cloud-integration-aws-new.md)`

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+For an existing AWS Account:
+
+* The customer must already have an AWS account that has been specifically mentioned in the AWS account transfer process. (Only approved accounts are authorized for this process.)
+
+* The customer must have reviewed the process for transferring an [existing Amazon Web Services account](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-existing/)
```
Comment (srveit) [inline Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Change link to `(partner-cloud-integration-aws-existing.md)`

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+* The customer has purchased AWS Reserved Instances or is thinking about buying AWS Reserved Instances.
+
+
+### Important Information
+
+#### Customer Reserved Instances
+
+Reserved Instances purchased prior to transferring to CenturyLink may still be used and must always stay with the account at which they were purchased.  If the Customer purchased Reserved Instances from a Master Payer they own, they can still use those Reserved Instances while under CenturyLink's care. Please review the process for migrating an [existing Amazon Web Services account](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-existing/), specifically the "Considerations" section.
```
Comment (srveit) [inline Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Change link to `(partner-cloud-integration-aws-existing.md)`

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+#### Customer Reserved Instances Monthly Increments that are not EC2
+AWS continues to expand the different products that are available to purchase Reserved Instances. CenturyLink accommodates this with any RI that is not an EC2 instance and receives monthly increment charges for those RI's will have the charges displayed on your bill with the line item description below.
+
+Line item description on your bill: Integrated, AWS Reserve Instances - SaaS
+
+
+### Contacting Cloud Application Manager Support
+
+We’re sorry you’re having an issue in [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/). Please review the [troubleshooting tips](https://www.ctl.io/knowledge-base/cloud-application-manager/troubleshooting/troubleshooting-tips/), or contact [Cloud Application Manager support](mailto:incident@CenturyLink.com) with details and screenshots where possible.
```
Comment (srveit) [inline Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Change second link to `(../Troubleshooting/troubleshooting-tips.md)`

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md (hunk 1 above)
Comment (amwatson1s) [reply Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Update completed.

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md (hunk 2 above)
Comment (amwatson1s) [reply Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Update completed.

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md (hunk 3 above)
Comment (amwatson1s) [reply Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Update completed.

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md (hunk 4 above)
Comment (amwatson1s) [reply Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Update completed.

File: Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md (hunk 5 above)
Comment (amwatson1s) [reply Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md]: Update completed.

**Pull request merged**